### DISCUSSION TITLE
Drag and drop to library / playlists

### DISCUSSION THREAD

**Discussion by EduMenges** [ciderapp/Cider#1102] [Feature Request]
One of the nicest things about Spotify is the ability to select a song, and simply drag and drop to a playlist. You can even create a new playlist with it, something that works for albums and singles as well.

![Spotify_Wd346fJoZz](https://user-images.githubusercontent.com/50274155/171440542-584b6ce1-2681-4ccc-a0ef-f84105fcbcfe.gif)

Apple Music doesn't have this feature. Instead, you have to use a menu and click to add to a playlist or to a library, something that is very time-consuming and annoying when you have a lot of playlists, and is one of the things that are preventing me from switching to it. For these reasons, I'm suggesting adding this option.

**Comment by vnxzd** [comment]
ITunes (the official frontend for apple music) has also this feature
![ezgif com-gif-maker](https://user-images.githubusercontent.com/93954853/173238186-0ac4e44d-69e2-46b2-8a9e-360c0d2c96cf.gif)
its pretty much confortable than right clicking a song and select the playlist
i also do suggest this feature for cider
